scripts/core/ui/components/Popup/Label.tsx

```diff
@@ -15,7 +15,7 @@ const Label: React.StatelessComponent<any> = ({text, children, centerText}) => (
         'popup__menu-label',
         {'popup__menu-label--center': centerText},
     )}>
-        {gettextCatalog(text)}
+        {gettext(text)}
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.

Reply:
I don't think it extracts that, the lib checks if the param is string literal

Reply:
If it doesn't extract it, there is no point of trying to translate it, since the translation wouldn't be found anyway. Unless the same key was translated previously, but that shouldn't be relied on. I see no reason why translated label cannot be passed as a prop.

Reply:
Ok so on all places where label is present I will add a call to gettext.
For example:

`<label className="sd-line-input__label">Annotation Body</label>`

will be changed to:

`<label className="sd-line-input__label">{gettext('Annotation Body')}</label>`

Reply:
`<label className="sd-line-input__label">{gettext('Annotation Body')}</label>` looks good, but I'm not sure if it's related to `Label.tsx`. When I lookup the usages of `Label.tsx` I get `scripts/core/ui/components/Form/ColouredValueInput/ColouredValuePopup.tsx` and `scripts/core/ui/components/Popup/Header.tsx`